### PR TITLE
Run ttm by systemd timer to prevent mem leaks

### PR DESCRIPTION
**Pull request by asdil12**

### CODE REVIEW

File: systemd/osrt-totest-manager@.service [hunk 1]
```diff
@@ -2,11 +2,8 @@
 Description=openSUSE Release Tools: ToTest Manager for %i
 
 [Service]
+Type=oneshot
```
Comment (jberry-suse) [inline systemd/osrt-totest-manager@.service]: I 100% agree with this based on documentation, but @lnussel ruled to use default (ie simple). So lets be consistent at the least. :)

File: systemd/osrt-totest-manager@.service [hunk 2]
```diff
@@ -2,11 +2,8 @@
...
 User=osrt-totest-manager
 SyslogIdentifier=osrt-totest-manager
 WorkingDirectory=~
-ExecStart=/usr/bin/osrt-totest-manager --verbose run --interval 5 "%i"
-
-[Install]
-WantedBy=multi-user.target
-
+ExecStart=/usr/bin/osrt-totest-manager --verbose run "%i"
```
Comment (jberry-suse) [inline systemd/osrt-totest-manager@.service]: Indeed. Suggested this for SLE repo-checker as well.

**Comment by jberry-suse** [comment]
See Travis output:

```
[   32s] error: Installed (but unpackaged) file(s) found:
[   32s]    /usr/lib/systemd/system/osrt-totest-manager@.timer
```

Needs to be in `dist/package/*.spec`.

**Comment by asdil12** [comment]
Added to spec

File: dist/package/openSUSE-release-tools.spec [hunk 3]
```diff
@@ -527,6 +527,7 @@ fi
 %{_bindir}/osrt-totest-manager
 %{_datadir}/%{source_dir}/totest-manager.py
 %{_unitdir}/osrt-totest-manager@.service
+%{_unitdir}/osrt-totest-manager@.timer
```
[the remark this answers is not in the document]
Comment (jberry-suse) [reply dist/package/openSUSE-release-tools.spec]: Does not bother me, although inconsistent with other files.

**Pull request merged**